clang/test/Driver/cxx23-modules-print-library-module-manifest-path.cpp

```diff
@@ -0,0 +1,40 @@
+// Test that -print-library-module-manifest-path finds the correct file.
+
+// RUN: rm -rf %t
```

Comment:
You can remove `mkdir -p %t` and simplify this to `rm -rf %t && split-file %s %t && cd %t`. This does place 3 commands on one line, but is pretty common.

Reply:
I copied the 3 line version from another test, I'll fix it.